examples/03-ui-components/12-custom-ui/App.tsx

```diff
@@ -28,30 +33,55 @@ export default function App() {
     ],
   });
 
+  // Automatically sets light/dark mode based on the user's system preferences.
+  const prefersDarkMode = useMediaQuery("(prefers-color-scheme: dark)");
+  const theme = useMemo(
+    () =>
+      createTheme({
+        palette: {
+          mode: prefersDarkMode ? "dark" : "light",
+        },
+      }),
+    [prefersDarkMode]
+  );
+
   // Renders the editor instance.
   return (
-    <BlockNoteView
-      editor={editor}
-      // Disables the UI components we want to replace.
-      formattingToolbar={false}
-      slashMenu={false}
-      sideMenu={false}>
-      {/* Adds the custom Formatting Toolbar */}
-      {/* `FormattingToolbarController isn't used since the custom toolbar is
-      static and always visible above the editor. */}
-      <CustomFormattingToolbar />
-      {/* Adds the custom Side Menu and Slash Menu. */}
-      {/* These use controllers since we want them to be positioned and
-      show/hide the same as the default ones.*/}
-      <SideMenuController sideMenu={CustomSideMenu} />
-      <SuggestionMenuController
-        triggerCharacter={"/"}
-        getItems={async (query) =>
-          filterSuggestionItems(getDefaultReactSlashMenuItems(editor), query)
-        }
-        suggestionMenuComponent={CustomSlashMenu}
-        onItemClick={(i) => i.onItemClick()}
-      />
-    </BlockNoteView>
+    // Provides theming for Material UI.
+    <ThemeProvider theme={theme}>
+      <BlockNoteViewRaw
+        editor={editor}
+        // Disables the UI components we'll be replacing.
+        formattingToolbar={false}
+        slashMenu={false}
+        sideMenu={false}
+        // Disables remaining UI elements.
+        linkToolbar={false}
+        filePanel={false}
+        tableHandles={false}
+        emojiPicker={false}>
+        {/* Adds the custom Formatting Toolbar. */}
+        {/* `FormattingToolbarController isn't used since the custom toolbar */}
+        {/* is static and always visible above the editor. */}
+        <MUIFormattingToolbar />
```

Comment:
Shall we also create a separate example with a static toolbar? (default mantine one?)